.gitignore

```diff
@@ -1,6 +1,8 @@
+.gradle
```

Comment:
If running `gradle` from top, this directory is created. Also, I think this will ignore .gradle in each sub-directory also, which is good, since it is a technical directory and mainly mustn't be committed.